functions/public/Invoke-WPFToggle.ps1

```diff
@@ -34,4 +34,6 @@ function Invoke-WPFToggle {
         "WPFToggleTaskbarAlignment" {Invoke-WinUtilTaskbarAlignment $(Get-WinUtilToggleStatus WPFToggleTaskbarAlignment)}
         "WPFToggleDetailedBSoD" {Invoke-WinUtilDetailedBSoD $(Get-WinUtilToggleStatus WPFToggleDetailedBSoD)}
     }
+
+    Invoke-WinUtilExplorerRefresh
```

Comment:
... isn't calling `Invoke-WinUtilExplorerRefresh` on every toggle interaction a bit over kill maybe? 😅
I think it's better to call it when necessary, and it'll show a clear dependency on this ExplorerRefresh function, which'll help with the `devdocs-generator` script.